ouroboros-consensus-cardano/ouroboros-consensus-cardano.cabal

```diff
@@ -167,3 +167,4 @@ executable db-validator
                        -fno-ignore-asserts
                        -threaded
                        -rtsopts
+                       "-with-rtsopts=-T -I0 -N2 -A16m"
```

Comment:
This is now same as cardano-node. It allows similar results, when the db-validator is used to count performance.